### ISSUE TITLE
[A11Y] Pages breaking around 360px width.

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1097]
## Finding

Technically not a conformance issue, but it seems layouts break apart around 360px width.

<img width="319" height="626" alt="" src="https://github.com/user-attachments/assets/92eef2f8-bfc3-420c-a4f6-27c0b554558e" />

## Location

Various pages

## Problems and solutions

Not everybody is free to pick any resolution when using a website. For compliance, everything should be working at 320px width. For eduid, it does seem like everything still works at this resolution, but it does fall apart it a bit. This has a negative impact on user confidence. And it's such a close call, there may be missed situations where functionality did degrade.

Make sure to design and test at 320px width.

## Resources
https://www.w3.org/WAI/WCAG22/Understanding/reflow.html
